tests/virtual_machines_test.go

```diff
@@ -52,10 +52,10 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("Should successfully update %s ConfigMap", names.WAITING_VMS_CONFIGMAP))
 		Expect(vmWaitConfigMap.Data).To(BeEmpty(), fmt.Sprintf("%s Data map should be empty", names.WAITING_VMS_CONFIGMAP))
 
-		// add vm opt-in label to the test namespaces
+		// remove all the labels from the test namespaces
```

Comment:
This is already in AfterEach, why do we neede it?

Reply:
The After each you are referring to does not apply to all the tests, but only to `Context: When the client creates a vm on a non-opted-out namespace`. There the `cleanNamespaceLabels` function has a specific role because we are testing to make sure opt-ed out vms still get kmp controller service (in order to remove their finalizers)

IIUC removing the labels in the "big" `BeforeEach()` is still needed.

Reply:
Done